v3io/aio/dataplane/kv.py

```diff
@@ -84,6 +85,8 @@ async def put(self,
             The access key with which to authenticate. Defaults to the V3IO_ACCESS_KEY env.
         condition (Optional) : str
             A Boolean condition expression that defines a conditional logic for executing the put-item operation.
+        update_mode (Optional) : str
+            CreateOrReplaceAttributes (default): Creates or replaces attributes
```

Comment:
i think `OverWriteAttributes` is the default for `PutItem`

Reply:
Ah

Reply:
I'll change to what you say, but looks like the whole field is undocumented, at least as of 2.8.